.github/workflows/release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  bumpversion:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.tag_version.outputs.new_version }}
+      previous_tag: ${{ steps.tag_version.outputs.previous_tag }}
+      bump_commit_sha: ${{ steps.bumpversion.outputs.commit_hash }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get next version
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
+          default_prerelease_bump: false
+          dry_run: true
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Create bumpversion
+        if: steps.tag_version.outputs.new_version
+        run: |
+          pip install bump2version
+          bump2version --new-version ${{ steps.tag_version.outputs.new_version }} setup.cfg
+      - name: Update Changelog
+        if: steps.tag_version.outputs.new_version
+        uses: stefanzweifel/changelog-updater-action@v1.6.2
+        with:
+          latest-version: ${{ steps.tag_version.outputs.new_tag }}
+          release-notes: ${{ steps.tag_version.outputs.changelog }}
+      - name: Commit bumpversion
+        id: bumpversion
+        if: steps.tag_version.outputs.new_version
+        uses: stefanzweifel/git-auto-commit-action@v4.14.1
+        with:
+          branch: ${{ github.ref }}
+          commit_message: "chore(release): preparing ${{ steps.tag_version.outputs.new_version }}"
```

Comment:
I don't think actions will run on this, but the commit message doesn't match the conventional commits and should just start with "chore:".

Reply:
Updated